CMake/AddMod2cSubmodule.cmake

```diff
@@ -16,24 +20,23 @@ if(NOT MOD2C_FOUND)
     message(FATAL_ERROR "git not found, clone repository with --recursive")
   endif()
   message(STATUS "Sub-module mod2c missing : running git submodule update --init --recursive")
-  execute_process(COMMAND ${GIT_EXECUTABLE} submodule update --init --recursive --
-                          ${CORENEURON_PROJECT_SOURCE_DIR}/external/mod2c
-                  WORKING_DIRECTORY ${CORENEURON_PROJECT_SOURCE_DIR})
+  execute_process(
+    COMMAND
+      ${GIT_EXECUTABLE} submodule update --init --recursive --
+      ${CORENEURON_PROJECT_SOURCE_DIR}/external/mod2c
+    WORKING_DIRECTORY ${CORENEURON_PROJECT_SOURCE_DIR})
 else()
   message(STATUS "Using mod2c submodule from ${MOD2C_PROJ}")
 endif()
 
 set(ExternalProjectCMakeArgs
-    -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE}
-    -DCMAKE_INSTALL_PREFIX=${CMAKE_BINARY_DIR}
+    -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE} -DCMAKE_INSTALL_PREFIX=${CMAKE_BINARY_DIR}
```

Comment:
Just a remark, old way was better to read

Reply:
200% agree and I complained about it multiple times 😅

Newer version of cmake-format has this issue. 

Only way is to downgrade version.

cc: @ohm314

Reply:
I agree, guys.. but can't do much about it. Let's not fret.